### PR TITLE
feat: Add SKUs for Condenser

### PR DESCRIPTION
**Pull request by bathomas**
- Maintain a list of SKUs in a config. map
- Provide data-only TF module
- Publish to GH pages

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,13 @@
...
-## ARC Terraform template
+## Condenser SKUs
 
-⚠️ This repository is still under construction! ⚠️
-
-A Terraform template for new ARC Terraform projects or modules. It has a
-suggested skeleton structure and GitHub Actions workflows.
+A config. map of Stock Keeping Numbers (SKUs) for objects on the Condenser
+platform. This config. map is used to generate a data-only Terraform module
+and static web page documentation, hosted at X.
```
Comment (brian-maher) [inline README.md]: Missing url? or not yet known?

File: web/scripts/yaml2html.sh [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  OS_DISK=$(yq ".data.skus.compute[] | select(.name == \"$SKU\") | .os_disk_gb" < $CONFIG_MAP_FILE)
+  GPU=$(yq ".data.skus.compute[] | select(.name == \"$SKU\") | .has_gpu" < $CONFIG_MAP_FILE)
+
+  echo "<tr>" >> table.tmp.html
+  echo "  <th scope=\"row\">$SKU</th>" >> table.tmp.html
+  echo "  <td>$VCPU</td>" >> table.tmp.html
+  echo "  <td>$RAM</td>" >> table.tmp.html
+  echo "  <td>$OS_DISK</td>" >> table.tmp.html
+  echo "  <td>$GPU</td>" >> table.tmp.html
+  echo "<tr>" >> table.tmp.html
```
Comment (brian-maher) [inline web/scripts/yaml2html.sh]: `</tr>`?

**Review by brian-maher — APPROVED** [review]

**Review by t-young31 — APPROVED** [review]
LGTM. Only suggestion: make this repo public, which is fine if the GH pages are(?)

File: README.md (hunk 1 above)
Comment (bathomas) [reply README.md]: Added. https://skus.condenser.arc.ucl.ac.uk (Liable to change ;) )

File: web/scripts/yaml2html.sh (hunk 2 above)
Comment (bathomas) [reply web/scripts/yaml2html.sh]: Done

**Comment by bathomas** [comment]
> LGTM. Only suggestion: make this repo public, which is fine if the GH pages are(?)

Done

**Pull request merged**